IPDatePicker/Classes/IPPickerComponentView.swift

```diff
@@ -74,37 +75,37 @@ final class IPTablePickerComponentView: UIView, IPPickerComponentView, UITableVi
 
     // MARK: - Selection
 
-    func setSelectedRow(_ row: Int, animated: Bool) {
-        selectedRow = row
-        centerOnRow(row, animated: animated)
+    func setSelectedItem(_ item: Int, animated: Bool) {
+        selectedItem = item
+        centerOnItem(item, animated: animated)
     }
 
-    // MARK: - UITableViewDataSource
+    // MARK: - InfiniteTableViewDataSource
 
     private let cellIdentifier = "cellIdentifier"
 
-    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    func numberOfItems(in infiniteTableView: InfiniteTableView) -> Int {
```

Comment:
Just curious - why isn't this `infiniteTableView(_ infiniteTableView: InfiniteTableView, numberOfItems items: Int)` to maintain as much parity with the original as possible AND with other protocol methods (like `infiniteTableView(_ infiniteTableView: InfiniteTableView, cellForItem item: Int, row: Int)`, directly below)?

Reply:
Never mind, I see what you're doing here.